readme/template-readme.md

```diff
@@ -0,0 +1,101 @@
+# README Template
+
+*{Before using this template, read the accompanying README template guide}.*
+
+## {Project} logo and badges
+{This section is optional. Share your logo and add relevant badges to your README.}
+
+## {Project Name} 
+{Optional: Include the project URL and the project owner name}
+
+## Table of contents 
+{This section is optional.}
+1. Heading 1
+2. Heading 2
+3. Heading 3...
+
+## Project description 
+{Project description example available in README template guide}
+
+With *{Project Name}* you can *(verb)* *(noun)*...
+
+*{Project Name}* helps you *(verb)* *(noun)*....
+
+Unlike *(alternative)*, *{Project Name}* *(verb)* *(noun)*... 
+
+{Optional: Include screenshots and/or demo videos}
+
+## Who this project is for 
+This project is intended for {target audience} who want to {audience objective}. 
+
+## Project dependencies
+To use {Project Name}, you will need: 
+- Prerequisite 1 
+- Prerequisite 2
+- Prerequisite 3...
+
+## Instructions for using {Project Name}
+Get started with {Project Name} by {first step here}
```

Comment:
**Suggestion (blocking, clarity):** I feel like I want more than just "first step here." Maybe take a look at Deanna's tutorial template and see how she formats steps in her template?

Edited to add: Actually, the way you did it in the next section was great. Maybe duplicate that in this section as well.